\documentclass[10pt,a4paper]{article}

\usepackage[a4paper,margin=0.7in]{geometry}

\usepackage[
    sorting=ydnt,
    maxnames=50,
    backend=biber,
    style=authoryear
]{biblatex}
\addbibresource{ref.bib}

% Slightly more space between bib items
\setlength{\bibitemsep}{1.5\itemsep}

% A bit more space around bib notes
\DeclareFieldFormat{note}{\vspace{0.5em}\par\small#1}

% Don't print "In: journal", just print "journal".
\renewbibmacro{in:}{%
  \ifentrytype{article}{}{%
    \printtext{\bibstring{in}\intitlepunct}%
  }%
}

\DeclareNameFormat{family}{%
  \namepartfamily
  \usebibmacro{name:andothers}%
}

\usepackage[english]{babel}

% Makes resume-specific commands available.
\usepackage{resume}

\begin{document}
\sloppy  % this to relax white spacing in favour of straight margins

\maintitle{David Pattinson}{September 19, 1989}{Updated \today}

\nobreakvspace{0.3em}  % add some page break averse vertical spacing

% \noindent prevents paragraph's first lines from indenting
% \mbox is used to obfuscate the email address
% \sbull is a spaced bullet
% \href well..
% \\ breaks the line into a new paragraph
\noindent\href{mailto:davipatti@protonmail.com}{davipatti\mbox{}@\mbox{}protonmail.com}\sbull
\textsmaller{+1} (516) 413-4078\sbull
\href{https://github.com/davipatti}{github.com/davipatti}
\par \noindent 1 Bungtown Rd, Cold Spring Harbor, NY 11724

\spacedhrule{0.5em}{-0.4em}  % a horizontal line with some vertical spacing before and after

\roottitle{Summary} \begin{multicols}{2}
  \emph{ I am a computational biologist with a background in evolutionary
  biology. My infectious disease research spans inferring infection histories from messy
  serological data, elucidating the antigenic effect of mutations and testing the
  predictability of seasonal influenza viruses. I was promoted to Senior Scientist after
  two years as a postdoc at the Influenza Research Institute (UW---Madison) and currently
  lead computational investigations in the group, teach, and run my own projects. In
  my career I have developed strong skills in Bayesian statistics, high throughput
  computing, data visualisation and phylogenetics. }

  \emph{
  Fundamentally, I am interested in how different aesthetic or mathematical
  representations of data can enable scientific inquiry. Aesthetically, I use data
  visualisation as a core component of my work, iterating to generate custom
  visualisations that provide detailed insight into datasets and the workings of
  statistical models whilst empowering the viewer to see patterns they didn't know they
  were looking for. Mathematically, I have used antigenic cartography and antibody
  landscapes heavily to elucidate antigenic variation of viruses and infer the effects of
  mutations.
  }
\end{multicols}

\spacedhrule{0.9em}{-0.4em}

\roottitle{Experience}

\headedsection
{Influenza Research Institute, {University of Wisconsin---Madison}}{\textsc{Madison WI, USA}}{

  \headedsubsection
  {Scientist I}
  {Mar \apo22--present}
  {

    \bodytext{ I was promoted to Senior Scientist after two years at the IRI and now lead
    computational investigations in the group comprising antigenic cartography, antibody
    landscapes, phylogenetics, Bayesian statistics, computational structural biology and
    next-generation sequencing analysis. }  

  }
  
  \headedsubsection
  {Postdoctoral Research Associate}
  {Nov \apo19--Mar \apo22}
  {

    \bodytext{I joined Prof.\@ Yoshihiro Kawaoka's lab to research mechanisms of
      antigenic change using computational structural biology. The COVID-19 pandemic
      prompted a switch to running a COVID-19 serosurveillance cohort in collaboration
      with {\href{https://www.marshfieldresearch.org/}{MCRI}} and the
      {\href{https://www.cdc.gov/}{U.S.\@ CDC.}} and investigating SARS-CoV-2 serology. }
  
  }
  
}

\spacedhrule{0.9em}{-0.4em}

\roottitle{Education}

\headedsection
{University of Cambridge}
{\textsc{Cambridge, UK}} {%
  \headedsubsection
  {Ph.D. Infectious Disease Informatics, Department of Zoology} {2014--2019}
  {\bodytext{Supervised by Prof.\@ Derek Smith.\sbull
  Quantifying the relationship between vaccine effectiveness and antigenic mismatch.\sbull
  LMMs for association testing and genotype-phenotype mapping applied to influenza invluenza virus antigenicity.\sbull
  A framework to predict influenza virus antigenic cluster transitions substitutions.
  } } }

\headedsection
{Imperial College London}
{\textsc{London, UK}} {%
  \headedsubsection
  {MRes Biosystematics, Natural History Museum}
  {2012--2013}
  {\bodytext{
        Endogenous retrovirus screening in catarrhine primates (Dr.\@ Michael Tristem).\sbull
        Novel methods in mitochondrial DNA enrichment (Dr.\@ Martijn Timmermans).\sbull
        A morphometric assessment of species delimitation in Canarian Pericallis (Dr.\@ Mark Carine).
    }
  }
}

\headedsection
{University of Cambridge}
{\textsc{Cambridge, UK}} {%
  \headedsubsection
  {BA Natural Science, Queens' College}
  {2009--2012}
  {
    \bodytext{
      Part II Zoology. Research Project: Combining molecular and
          morphological data in phylogenetic analyses (Dr.\@ Robert Asher).\sbull
      Part IB Biochemistry, Animal Biology, Plant Biology.\sbull
      Part IA Evolution and Behaviour, Biology of Cells, Chemistry,
          Mathematics for Biologists.
    }
  }
}

\spacedhrule{0.9em}{-0.4em}

\roottitle{Skills}

\inlineheadsection  % special section that has an inline header with a 'hanging' paragraph
{Computational:} {
  
  I have used \textbf{python} virtually every day since 2013; developing (e.g.\@
  \href{https://ititer.readthedocs.io/}{ititer},
  \href{https://pymds.readthedocs.io}{pymds}) and using scientific research packages
  (e.g. \textbf{pymc}, \textbf{numpy}, \textbf{MDAnalysis}, \textbf{scipy},
  \textbf{scikit-learn}, \textbf{matplotlib}, \textbf{pandas}, \textbf{bambi}). I make
  interactive visualisation dashboards with \textbf{dash}.\sbull I am a proponent of
  \textbf{unit testing} and \textbf{reproducible research practices}.\sbull I am familiar
  with \textbf{R} and \textbf{JavaScript}, and am learning \textbf{Rust}. \sbull I use
  the unix command line and \textbf{git} daily.\sbull Bioinformatics software I have used
  includes: \textbf{MrBayes}, \textbf{RAxML} and \textbf{mafft} for phylogenetics and
  \textbf{gromacs} and \textbf{amber} for structural biology, among many others.\sbull I
  have used \textbf{snakemake} to automate pipelines in several projects.\sbull I have
  used the \textbf{slurm} and \textbf{HTCondor} cluster submission systems.}

\vspace{0.5em}
\inlineheadsection
{Natural languages:}
{English \emph{(mother tongue)}, German\emph{(elementary)}.}

\spacedhrule{1.6em}{-0.4em}

\nocite{*}
\printbibliography[title=Publications]

\spacedhrule{1.6em}{-0.4em}

\end{document}

