% LaTeX source of my resume
% =========================

% Commented for easy reuse... ;)

% See the `README.md` file for more info.

% This file is licensed under the CC-NC-ND Creative Commons license.


% Start a document with the here given default font size and paper size.
\documentclass[10pt,a4paper]{article}

% Set the page margins.
\usepackage[a4paper,margin=0.7in]{geometry}

% For publications
\usepackage[square,numbers]{natbib}
\bibliographystyle{abbrvnat}

% Setup the language.
\usepackage[english]{babel}
\hyphenation{Some-long-word}

% Makes resume-specific commands available.
\usepackage{resume}

\begin{document}  % begin the content of the document
\sloppy  % this to relax whitespacing in favour of straight margins

% title on top of the document
\maintitle{David Pattinson}{September 19, 1989}{Last update on \today}

\nobreakvspace{0.3em}  % add some page break averse vertical spacing

% \noindent prevents paragraph's first lines from indenting
% \mbox is used to obfuscate the email address
% \sbull is a spaced bullet
% \href well..
% \\ breaks the line into a new paragraph
\noindent\href{mailto:davipatti@protonmail.com}{davipatti\mbox{}@\mbox{}protonmail.com}\sbull
\textsmaller{+1} (516) 413-4078\sbull
\href{https://github.com/davipatti}{github.com/davipatti}
\\
1 Bungtown Rd, Cold Spring Harbor, NY 11724

\spacedhrule{0.5em}{-0.4em}  % a horizontal line with some vertical spacing before and after

\roottitle{Summary}  % a root section title

\begin{multicols}{2}
  \noindent \emph{I am a computational scientist with a strong background in evolutionary
  biology and have worked at the intersection of virology, immunology and disease
  informatics since I began my Ph.D. in 2014. My current research focuses on assessing
  and predicting seasonal influenza virus antigenic evolution as well as tracking
  population immune responses to SARS-CoV-2. I use a wide range of methods including
  Bayesian data analysis, antigenic cartography, antibody landscapes, phylogenetics and
  computational structural biology. I enjoy creating custom data visualisations to gain
  deep insights into patterns in data, underlying processes and how models function.}
\end{multicols}

\spacedhrule{0.9em}{-0.4em}

\roottitle{Experience}

\headedsection
{Influenza Research Institute, {\href{https://www.wisc.edu/}{University of Wisconsin--Madison}}}{\textsc{Madison WI, USA}}{

  \headedsubsection
  {Scientist I}
  {Mar \apo22 -- present}
  {

    \bodytext{ I was promoted to Scientist two years after joining Prof.\@ Kawaoka's lab
      having provided expertise in antigenic cartography, antibody landscapes,
      phylogenetics, computational structural biology, next-generation sequencing
      analysis to the group. I teach in small groups, write bespoke analysis pipelines,
      and continue my own research projects.}  

  }
  
  \headedsubsection
  {Postdoctoral Research Associate}
  {Nov \apo19 -- Mar \apo22}
  {

    \bodytext{I joined Prof.\@ Yoshihiro Kawaoka's lab to research
      mechanisms of antigenic change using computational structural biology. The
      COVID-19 pandemic prompted a switch to studying SARS-CoV-2 serology, for
      instance by estimating important epidemioloical parameters such as
      {\href{https://www.sciencedirect.com/science/article/pii/S2589537021000146}{antibody
      deccay rates}}. I helped plan, design experiments for, and conduct analyses
      for a prospective serosurveillance cohort in collaboration with
      {\href{https://www.marshfieldresearch.org/}{Marshfield Clinic Research
      Institute}} and the {\href{https://www.cdc.gov/}{U.S.\@ CDC.}}}
  
  }
  
}

\spacedhrule{0.9em}{-0.4em}

\roottitle{Education}

\headedsection
{University of Cambridge}
{\textsc{Cambridge, UK}} {%
  \headedsubsection
  {Ph.D. Infectious Disease Informatics, Department of Zoology} {2014 -- 2019}
  {\bodytext{Supervised by Prof.\@ Derek Smith. Quantifying the relationship between
  vaccine effectiveness and antigenic mismatch. LMMs for association testing and
  GP-mapping with influenza antigenic phenotypes. A framework to rank substitutions by
  similarity to those that have caused antigenic cluster transitions.} } }

\headedsection
{Imperial College London}
{\textsc{London, UK}} {%
  \headedsubsection
  {MRes Biosystematics, Natural History Museum}
  {2012 -- 2013}
  {\bodytext{
        Endogenous retrovirus screening in catarrhine primates (Dr.\@ Michael Tristem).\sbull
        Novel methods in mitochondrial DNA enrichment (Dr.\@ Martijn Timmermans).\sbull
        A morphometric assessment of species delimitation in Canarian Pericallis (Dr.\@ Mark Carine).
    }
  }
}

\headedsection
{University of Cambridge}
{\textsc{Cambridge, UK}} {%
  \headedsubsection
  {BA Natural Science, Queens' College}
  {2009 -- 2012}
  {
    \bodytext{
      Part II Zoology. Research Project: Combining molecular and
          morphological data in phylogenetic analyses (Dr.\@ Robert Asher).\sbull
      Part IB Biochemistry, Animal Biology, Plant Biology.\sbull
      Part IA Evolution and Behaviour, Biology of Cells, Chemistry,
          Mathematics for Biologists.
    }
  }
}

\headedsection
{Aylesbury Grammar School}
{\textsc{Aylesbury, UK}} {
  \headedsubsection
  {A level Maths, Further Maths, Biology, Chemistry, Physics}
  {2000 -- 2008}
  {}
}

\spacedhrule{0.9em}{-0.4em}

\roottitle{Skills}

\inlineheadsection  % special section that has an inline header with a 'hanging' paragraph
{Computational:} {
  
  I have used \textbf{python} daily since 2013; developing (e.g.\@
  \href{https://ititer.readthedocs.io/}{ititer},
  \href{https://pymds.readthedocs.io}{pymds}) and using scientific research packages
  (e.g. \textbf{pymc}, \textbf{numpy}, \textbf{MDAnalysis}, \textbf{scipy},
  \textbf{scikit-learn}, \textbf{matplotlib}, \textbf{pandas}, \textbf{bambi}). I make
  interactive visualisation dashboards with \textbf{dash}.\sbull I am a proponent of
  \textbf{unit testing} and \textbf{reproducible research practices}.\sbull I am familiar
  with \textbf{R} and \textbf{JavaScript}, and am learning \textbf{Rust}. \sbull I use
  the unix command line and \textbf{git} daily.\sbull Bioinformatics software I have used
  includes: \textbf{MrBayes}, \textbf{RAxML} and \textbf{mafft} for phylogenetics and
  \textbf{gromacs} and \textbf{amber} for structural biology, among many others.\sbull I
  have used \textbf{snakemake} to automate pipelines in several projects.\sbull
  I have used the \textbf{slurm} and \textbf{HTCondor} cluster submission systems.}

\vspace{0.5em}
\inlineheadsection
{Natural languages:}
{English \emph{(mother tongue)} and German\emph{(elementary)}.}


\spacedhrule{1.6em}{-0.4em}

\roottitle{Publications}

\inlineheadsection
{}
{
  \renewcommand{\bibsection}{
    % (See this list on \href{https://scholar.google.co.uk/citations?user=q260RVcAAAAJ&hl=en}{Google scholar}.)
}
  \nocite{*}
  \bibliography{ref}
}

\spacedhrule{1.6em}{-0.4em}

% \roottitle{Interests}

% \inlineheadsection
%   {Non-exhaustive and in alphabetical order:}
%   {art, Buddhism, cryptography, functional programming, Go (board game), history, music (from classical and jazz to Berlin-techno), \acr{NLP}, permaculture, philosophy, rock climbing, startups, travel, typography (e.g.\ graphic design, \LaTeX), \acr{UX}-design and vegan cuisine.}

\end{document}

